license.txt

```diff
@@ -0,0 +1,12 @@
+
+
+<!--
+Copyright (c) 2018 by freeCodeCamp (https://codepen.io/freeCodeCamp/pen/MJjpwO)
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Kindly add this file to .gitignore
this file is not meant to be shown

Reply:
- [ ] I think we can improve this comment message review by explaining why this change is required as you did for other comments and if possible try to use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

Reply:
- [ ] I think it would be better to improve your summary:
```bash
No linters error
nice commit messages
good HTML semantic use
Nice Styling! ;)
....
```
by editing it and making it more professional and human-friendly. I suggest you use one of the previous summaries you received on your reviews or even use the current one according to the status of your review i.e if it is `Required` use the required template and if it is `Approved` you can use the approved one.

Reply:
@tresorsawasawa alright i guess it should be explained,
tho about the emojis part where do find them ..

Reply:
Hi @ChaabaneAnas 
Just click [here](https://www.youtube.com/watch?v=k899Tn_eiYk) and how to add emojis in a GitHub comment